.github/workflows/sync-upstream-change.yml

```diff
@@ -0,0 +1,94 @@
+name: Sync with Upstream (change)
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    if: github.event.review.state == 'changes_requested' && startsWith(github.event.pull_request.head.ref, 'sync-upstream-')
```

Comment:
How about looking at the `body` too, so that this doesn't trash the work if someone leaves a `changes_requested` review on a manual resolution?

So you can do something like

```
@bot rebase
```

and it does it.  Like what you have with Dependabot and Atlantis.

Instructions would need to be added to the template tho.

Reply:
We are not rebasing anymore.. I'm thinking of going with `bot remerge`.

Reply:
Done!

Reply:
We aren't but the action the bot would do would be to rebase its changes (i.e. do them again)

anyway `update` is fine 👍!